pkg/collector/corechecks/oracle-dbm/sql/setup.sql

```diff
@@ -1,82 +1,58 @@
-set verify off
-
-accept password hide prompt password:
-
-CREATE USER c##datadog IDENTIFIED BY &password CONTAINER = ALL ;
-
-ALTER USER c##datadog SET CONTAINER_DATA=ALL CONTAINER=CURRENT;
-
-grant create session to c##datadog ;
-grant select on v_$session to c##datadog ;
-grant select on v_$database to c##datadog ;
-grant select on v_$containers to c##datadog;
-grant select on v_$sqlstats to c##datadog ;
-grant select on v_$instance to c##datadog ;
-grant select on v_$sql_plan_statistics_all to c##datadog ;
-grant select on dba_feature_usage_statistics to c##datadog ;
-grant select on v_$datafile to c##datadog ;
-grant select on v_$con_sysmetric to c##datadog ;
-grant select on v_$sysmetric to c##datadog ;
-grant select on cdb_tablespace_usage_metrics to c##datadog ;
-grant select on cdb_tablespaces to c##datadog ;
-grant select on v_$process to c##datadog ;
-grant select on v_$sgainfo to c##datadog ;
-
 CREATE OR REPLACE VIEW dd_session AS
-SELECT 
-    s.indx as sid,
-    s.ksuseser as serial#,
-    s.ksuudlna as username,
-    DECODE(BITAND(s.ksuseidl, 9), 1, 'ACTIVE', 0, DECODE(BITAND(s.ksuseflg, 4096), 0, 'INACTIVE', 'CACHED'), 'KILLED') as status,
-    s.ksuseunm as osuser,
-    s.ksusepid as process,
-    s.ksusemnm as machine,
-    s.ksusemnp as port,
-    s.ksusepnm as program,
-    DECODE(BITAND(s.ksuseflg, 19), 17, 'BACKGROUND', 1, 'USER', 2, 'RECURSIVE', '?') as type,
-    s.ksusesqi as sql_id,
-    sq.force_matching_signature as force_matching_signature,
-    s.ksusesph as sql_plan_hash_value,
-    s.ksusesesta as sql_exec_start,
-    s.ksusesql as sql_address,
-    CASE WHEN BITAND(s.ksusstmbv, POWER(2, 04)) = POWER(2, 04) THEN 'Y' ELSE 'N' END as in_parse,
-    CASE WHEN BITAND(s.ksusstmbv, POWER(2, 07)) = POWER(2, 07) THEN 'Y' ELSE 'N' END as in_hard_parse,
-    s.ksusepsi as prev_sql_id,
-    s.ksusepha as prev_sql_plan_hash_value,
-    s.ksusepesta as prev_sql_exec_start,
-    sq_prev.force_matching_signature as prev_force_matching_signature,
-    s.ksusepsq as prev_sql_address,
-    s.ksuseapp as module,
+SELECT
```

Comment:
Do we need to update our setup instructions based on this change?

Reply:
Yes - I have a separate ticket for that: https://datadoghq.atlassian.net/browse/DBMON-2789